DEVELOPERS.md

```diff
@@ -35,10 +50,14 @@ Note: just commands will only work inside git-bash shell by default, as they ass
 
 ## Local development environment
 
-Set up a local development environment with:
+Run the Django devserver with:
+```
+just run
```

Comment:
I remember needing to run `just assets` before this command would work, because this doesn't trigger them.

This might be fine if you're running the Vite devserver, but that should be optional.

Reply:
It's got `assets` as a dependency so that step isn't necessary now 🎉